src/chariot/database/MongoDatabase.py

```diff
@@ -0,0 +1,50 @@
+from tinymongo import TinyMongoClient
+import os
+from datetime import datetime
+from typing import List
+import shutil
+
+from chariot.database.Database import Database
+
+
+class MongoDatabase(Database):
+    def __init__(self, db_path: str = '../data/iot_database'):
+        Database.__init__(self, db_path=db_path)
+        self.connectToDB()
+
+    def initializeDB(self):
+        self.database = self.conn.database
+        self.collection = self.database.collection
+
+    def connectToDB(self):
+        # Make the data directory if it does not exist
+        try:
+            os.makedirs(self.db_path)
+        except FileExistsError:
+            pass
+        os.chmod(self.db_path, 0o700)
+
+        self.conn = TinyMongoClient(self.db_path)
```

Comment:
In general, it seems you are not adhering to our use of type hints. All instance variables should be defined and typed in the constructor. Try running the mypy tool on each of these files and fix the errors that come up.